### PR TITLE
Add cypress dashboard key

### PR DESCRIPTION
**Pull request by hvuong-sigma**

### CODE REVIEW

**Review by JSchwerberg — CHANGES_REQUESTED** [review]
Looks like the auto-formatter added some bad indents here, please fix.

File: job-templates/job.yaml [hunk 1]
```diff
@@ -13,21 +13,30 @@ spec:
...
       - name: buildkite-agent
         image: buildkite/agent:latest
-        args: ["start", "--disconnect-after-job", "--disconnect-after-job-timeout", "300"]
+        args:
+          [
+            "start",
+            "--disconnect-after-job",
+            "--disconnect-after-job-timeout",
+            "300",
+          ]
```
Comment (LiuVII) [inline job-templates/job.yaml]: this is like another commit, but also why?

File: job-templates/job.yaml [hunk 2]
```diff
@@ -13,21 +13,30 @@ spec:
...
         imagePullPolicy: Always
         securityContext:
           privileged: true
         env:
           - name: TERM
             value: dumb
           - name: BUILDKITE_AGENT_TOKEN
-            valueFrom: {secretKeyRef: {name: buildkite-agent, key: token}}
+            valueFrom: { secretKeyRef: { name: buildkite-agent, key: token } }
```
Comment (LiuVII) [inline job-templates/job.yaml]: same

File: job-templates/job.yaml [hunk 3]
```diff
@@ -55,17 +64,21 @@ spec:
...
-          secret:
-            secretName: buildkite-agent-ssh
-            defaultMode: 0400
-        - name: yarn-cache
-          hostPath: {path: /tmp/yarn-cache}
-        - name: docker-binary
-          hostPath: {path: /usr/bin/docker}
-        - name: docker-socket
-          hostPath: {path: /var/run/docker.sock}
+      - name: browserstack-tests-ssh-keys
```
Comment (LiuVII) [inline job-templates/job.yaml]: same another commit

**Review by LiuVII — COMMENTED** [review]
Should split refactoring commit and adding key commit, refactoring goes first

File: job-templates/job.yaml [hunk 4]
```diff
@@ -13,21 +13,25 @@ spec:
...
+          ["start", "--disconnect-after-job", "--disconnect-after-job-timeout", "300"]
         imagePullPolicy: Always
         securityContext:
           privileged: true
         env:
           - name: TERM
             value: dumb
           - name: BUILDKITE_AGENT_TOKEN
-            valueFrom: {secretKeyRef: {name: buildkite-agent, key: token}}
+            valueFrom: { secretKeyRef: { name: buildkite-agent, key: token } }
```
Comment (JSchwerberg) [inline job-templates/job.yaml]: Remove new whitespace

File: job-templates/job.yaml [hunk 5]
```diff
@@ -13,21 +13,25 @@ spec:
       containers:
       - name: buildkite-agent
         image: buildkite/agent:latest
-        args: ["start", "--disconnect-after-job", "--disconnect-after-job-timeout", "300"]
+        args:
+          ["start", "--disconnect-after-job", "--disconnect-after-job-timeout", "300"]
```
Comment (JSchwerberg) [inline job-templates/job.yaml]: Remove new linebreak

**Review by JSchwerberg — CHANGES_REQUESTED** [review]

**Review by JSchwerberg — APPROVED** [review]

File: job-templates/job.yaml (hunk 1 above)
Comment (hvuong-sigma) [reply job-templates/job.yaml]: sorry, it was my autoformatter - should have double checked. fixed though!

File: job-templates/job.yaml (hunk 2 above)
Comment (hvuong-sigma) [reply job-templates/job.yaml]: fixed

File: job-templates/job.yaml (hunk 3 above)
Comment (hvuong-sigma) [reply job-templates/job.yaml]: fixed

**Pull request merged**